README-sycl.md

```diff
@@ -52,6 +52,7 @@ ninja [EXAMPLE_NAME]_sycl
 You can run it like this from your build directory
 
 ```
+LD_LIBRARY_PATH=/path/to/sycl/install/lib 
```

Comment:
if you split it over two lines you need export, don't you?
```suggestion
export LD_LIBRARY_PATH=/path/to/sycl/install/lib 
```

Note that you previously had it on the same line in
https://github.com/codeplaysoftware/cutlass-fork/pull/18/files
but then 
https://github.com/codeplaysoftware/cutlass-fork/pull/20 removed it before it was added back in here. 

(which made reviewing https://github.com/codeplaysoftware/cutlass-fork/pull/26 really confusing)

Reply:
Thanks @rolandschulz  for catching this. I did not realised that it has been broken over the line. Fixed it now in the new PR 
https://github.com/codeplaysoftware/cutlass-fork/pull/31

Reply:
Yes, it should use 'export' if on a different line. Good catch, I'll create a PR for that.